manifests/config.pp

```diff
@@ -2,15 +2,24 @@
 class ntp::config inherits ntp {
 
   if $keys_enable {
-    $directory = ntp_dirname($keys_file)
-    file { $directory:
-      ensure  => directory,
-      owner   => 0,
-      group   => 0,
-      mode    => '0755',
+    file { $keys_file:
```

Comment:
I would've assumed the file being managed previously (which is now being corrected). So indeed it actually is backwards incompatible. But we aren't replacing the keys file here (don't see a file or template directive), only modifying the owner/group/permission set, so it seems trivial to me?